gateway-ha/src/main/java/io/trino/gateway/ha/clustermonitor/TrinoHealthStateType.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.gateway.ha.clustermonitor;
+
+/**
+ * PENDING is for ui/observability purpose and functionally it's unhealthy
+ * We should use PENDING when Trino clusters are still spinning up
+ * HEALTHY is when health checks report clusters as up
+ * UNHEALTHY is when health checks report clusters as down
```

Comment:
How about changing HEALTHY/UNHEALTHY to UP/DOWN?

Reply:
I think UP/DOWN has the implication of server being up or down. HEALTHY could be a better candidate here because it means it passed the healthcheck. On the other hand, UNHEALTHY means it failed the healthcheck, but the trino cluster is still UP